packs/hello-universe-1.1.2/values.yaml

```diff
@@ -0,0 +1,16 @@
+# spectrocloud.com/enabled-presets: Backend:disable-api
+
+pack:
+  content:
+    images:
+      - image: ghcr.io/spectrocloud/hello-universe:1.1.2
+
+
+manifests:
+  hello-universe:
+    images:
+      hellouniverse: ghcr.io/spectrocloud/hello-universe:1.1.2
+    apiEnabled: false
+    namespace: hello-universe
+    port: 8080
```

Comment:
We should also expose the API port and leave with a default of 3000. Just in case it may conflict with someone's applications on the cluster.

Reply:
As discussed, we cannot change this. I have added it under prerequisites in the README to ensure users don't miss it.